src/media/mod.rs

```diff
@@ -253,6 +253,7 @@ impl Media {
                         contiguous: dep.contiguous,
                         ext_vals: dep.ext_vals().clone(),
                         codec_extra: dep.codec_extra,
+                        last_sender_info: dep.first_sender_info(),
```

Comment:
Here where you are composing MediaData can you get the last SR from the stream instead getting the potentially outdated one from dep ? (https://github.com/algesten/str0m/pull/333/files#r1318203310)

Reply:
Yes we could, but I think we want to outdated one i.e the one that applies to the timestamps in this sample

Reply:
As I explained in https://github.com/algesten/str0m/pull/333#discussion_r1318289998 it's intentionally done like this